pkgs/by-name/af/affine/package.nix

```diff
@@ -35,212 +35,205 @@ let
   nodejs = nodejs_20;
   yarn = yarn-berry.override { inherit nodejs; };
 in
-stdenv.mkDerivation (
-  finalAttrs:
-  (
-    {
-      productName = if buildType == "stable" then "AFFiNE" else "AFFiNE-" + buildType;
-      binName = lib.toLower finalAttrs.productName;
-      pname = finalAttrs.binName;
-
-      # https://github.com/toeverything/AFFiNE/releases/tag/v0.18.1
-      version = "0.18.1";
-      GITHUB_SHA = "8b066a4b398aace25a20508a8e3c1a381721971f";
-      src = fetchFromGitHub {
-        owner = "toeverything";
-        repo = "AFFiNE";
-        rev = finalAttrs.GITHUB_SHA;
-        hash = "sha256-TWwojG3lqQlQFX3BKoFjJ27a3T/SawXgNDO6fP6gW4k=";
-      };
-
-      meta =
-        {
-          description = "Workspace with fully merged docs, whiteboards and databases";
-          longDescription = ''
-            AFFiNE is an open-source, all-in-one workspace and an operating
-            system for all the building blocks that assemble your knowledge
-            base and much more -- wiki, knowledge management, presentation
-            and digital assets
-          '';
-          homepage = "https://affine.pro/";
-          license = lib.licenses.mit;
-          maintainers = with lib.maintainers; [ xiaoxiangmoe ];
-          platforms = [
-            "aarch64-darwin"
-            "aarch64-linux"
-            "x86_64-darwin"
-            "x86_64-linux"
-          ];
-          sourceProvenance = [ lib.sourceTypes.fromSource ];
-        }
-        // lib.optionalAttrs hostPlatform.isLinux {
-          mainProgram = finalAttrs.binName;
```

Comment:
In linux we have mainProgram but in macOS we have no this.
Should we keep this or remove this?

Reply:
In macos this is not correct